react-client/src/index.jsx

```diff
@@ -1,4 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Slider from './components/Slider.jsx';
+import axios from 'axios';
+import { useParams } from 'react-router';
 
-ReactDOM.render(<h1>Hello From REACT</h1>, document.getElementById('app'));
\ No newline at end of file
+
+let getImagesById =  ()=>{
+  let url = window.location.href;
+  url = url.split('/');
+  console.log('URL, ', url);
+  let id = url[4];
+
+  axios.get(`/item/${id}/images`)
+  .then((res)=>{
+    console.log('DATA ', res.data);
+    ReactDOM.render(<Slider slides={res.data}/>, document.querySelector('body'));
```

Comment:
you might want to choose a different selector so that it's unique for when others pull down your code

Reply:
Changed this to `document.getElementById('images')` and added a div in my index,html file